.github/workflows/nightly-e2e.yml

```diff
@@ -327,11 +327,8 @@ jobs:
       - partial-set-security-validators-power-cap-test
       - partial-set-security-validators-allowlisted-test
       - partial-set-security-validators-denylisted-test
-<<<<<<< HEAD
       - active-set-changes-test
-=======
       - partial-set-security-modification-proposal
```

Comment:
Would be nice if this was under `partial-set-security-validators-denylisted-test` so all the `partial-set-security-*` tests are together.